GazePianoPrototype/MainPage.xaml

```diff
@@ -36,21 +36,21 @@
                     <RowDefinition Height="*" />
                 </Grid.RowDefinitions>
                 <TextBlock Grid.Row="0" Foreground="#00FFFF" Text="Play Music" FontSize="48" HorizontalAlignment="Center" />
-                <HyperlinkButton Grid.Row="1" Background="#4A86E8" Click="HyperlinkButton_Click" Margin="150,40,150,50"
+                <Button Grid.Row="1" Background="#4A86E8" Click="HyperlinkButton_Click" Margin="150,40,150,50"
                                  gaze:GazeInput.DwellDuration="00:00:00.500" HorizontalAlignment="Stretch" VerticalAlignment="Stretch">
```

Comment:
Since we're using the community toolkit, we can get rid of a lot of the extra code on this page that isn't doing much